src/mame/igs/pgm.cpp

```diff
@@ -5721,3 +5864,6 @@ GAME( 200?, kovlsqh2,     kovshp,    pgm_arm_type1,          kovsh,     pgm_arm_
 //乱世街霸/Luànshì jiē bà
 GAME( 200?, kovlsjb,      kovshp,    pgm_arm_type1,          kovsh,     pgm_arm_type1_state, init_kovlsqh2, ROT0,   "bootleg", "Luanshi Jie Ba (bootleg of Knights of Valour Super Heroes Plus, ver. 200CN, set 1)", MACHINE_IMPERFECT_SOUND | MACHINE_UNEMULATED_PROTECTION | MACHINE_NOT_WORKING | MACHINE_SUPPORTS_SAVE ) /* need internal rom of IGS027A */
 GAME( 200?, kovlsjba,     kovshp,    pgm_arm_type1,          kovsh,     pgm_arm_type1_state, init_kovlsqh2, ROT0,   "bootleg", "Luanshi Jie Ba (bootleg of Knights of Valour Super Heroes Plus, ver. 200CN, set 2)", MACHINE_IMPERFECT_SOUND | MACHINE_UNEMULATED_PROTECTION | MACHINE_NOT_WORKING | MACHINE_SUPPORTS_SAVE ) /* need internal rom of IGS027A */
+
+// Rock Fever 1
+GAME ( 1999, rf1, pgm, pgm_arm_type1_rf, pgm, pgm_arm_type1_state, init_rf1, ROT0, "IGS", "Rock Fever 1", MACHINE_IS_SKELETON ) /* need internal rom of IGS027A */
```

Comment:
Is it really titled "Rock Fever 1" or just "Rock Fever"? Flyers I've seen around seem to call it just "Rock Fever". This video also shows only Rock Fever.
Also, as Haze said, the comment at this end should be updated as it doesn't just need the internal ROM of the IGS027A, but also dumps of most of the ROMs.